scripts/globals/effects/defender.lua

```diff
@@ -7,16 +7,18 @@ require("scripts/globals/status")
 -----------------------------------
 
 function onEffectGain(target,effect)
-    target:addMod(tpz.mod.DEFP,25)
-    target:addMod(tpz.mod.RATTP,-25)
-    target:addMod(tpz.mod.ATTP,-25)
+    local power = effect:getPower()
```

Comment:
What is effect:getPower() expected to return here? Is this just modernizing? Defender is a static 25% as far as I know.